src/main/java/org/fundacionjala/virtualassistant/service/ASRClient.py

```diff
@@ -0,0 +1,20 @@
+import os
+import whisper
+from flask import Flask, request
+
+app = Flask(__name__)
+
+@app.route('/record', methods=['POST'])
+def convertToText():
```

Comment:
this function does is breaking SOLID principles because you are saving, loading, and transcribing in only a function please break down this funtion

Reply:
Dont use  "No audio file found" in the response, better is to define and enum with the possible values